### PR TITLE
Add workflow to run scenarios

### PR DESCRIPTION
**Pull request by christopherwoodall**
Adds a workflow to run scenarios for a given eval period to 0.16.1 release 

### changes outside of CI and test
- In `armory/docker/management.py` replace decode errors from armory output; there are ANSI line-drawing characters which are not convertable
- In `armory/__main__.py` function `run` now returns a status code to `main` instead of `sys.exit(code)`

### CODE REVIEW

File: armory/__main__.py [hunk 1]
```diff
@@ -668,9 +668,8 @@ def launch(command_args, prog, description):
     _set_gpus(config, args.use_gpu, args.no_gpu, args.gpus)
     (config, args) = arguments.merge_config_and_args(config, args)
 
+    # this is the expected meaning of `launch()` that is, start an interactive session even if `--interactive` was not specified
     rig = Evaluator(config, root=args.root)
-    if not args.interactive and not args.jupyter:
-        args.interactive = True
```
Comment (mwartell) [inline armory/__main__.py:673]: This conditional needs to remain.

File: .github/workflows/1-scan-lint-build.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
+# yamllint disable
```
Comment (mwartell) [inline .github/workflows/1-scan-lint-build.yml:2]: All `# yamllint` flags need to die.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -44,5 +40,4 @@ body:
       label: Additional Information
       description: |
         Include any additional information such as logs, screenshots, or scenarios in which the bug occurs in order to facilitates resolving the issue.
-
 # yamllint enable rule:line-length
```
Comment (mwartell) [inline .github/ISSUE_TEMPLATE/bug_report.yml:43]: kill `# yamllint`

**Review by mwartell — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (christopherwoodall) [reply .github/ISSUE_TEMPLATE/bug_report.yml:43]: Will address in https://github.com/twosixlabs/armory/pull/1708

File: armory/__main__.py (hunk 1 above)
Comment (christopherwoodall) [reply armory/__main__.py]: Reverted

**Review by mwartell — APPROVED** [review]

**Pull request merged**